src/components/qualityOptions.js

```diff
@@ -32,29 +75,57 @@ export function getVideoQualityOptions(options) {
 
     // Quality options are indexed by bitrate. If you must duplicate them, make sure each of them are unique (by making the last digit a 1)
     if (maxAllowedWidth >= 3800 && hostScreenWidth >= 1930) {
-        qualityOptions.push({ name: '4K - 120 Mbps', maxHeight: 2160, bitrate: 120000000 });
-        qualityOptions.push({ name: '4K - 80 Mbps', maxHeight: 2160, bitrate: 80000000 });
+        qualityOptions2160p.forEach(function(qualityOption) {
+            if (videoBitRate) {
+                if (qualityOption.bitrate < videoBitRate * maxStreamingBitrateMultiplier) {
```

Comment:
By running the `videoBitRate * maxStreamingBitrateMultiplier` calculation in all these forEach functions you're calculating the same value 24 times. Better to move it outside of these loops and only calculate it once.